com.unity.render-pipelines.high-definition/Editor/RenderPipeline/CustomPass/CustomPassVolumeEditor.cs

```diff
@@ -235,6 +235,21 @@ void CreateReorderableList(SerializedProperty passList)
                 }
                 menu.ShowAsContext();
 			};
+
+            m_CustomPassList.onReorderCallback = (index) => ClearCustomPassCache();
+
+            m_CustomPassList.onRemoveCallback = (list) =>
+            {
+                ReorderableList.defaultBehaviours.DoRemoveButton(list);
+                ClearCustomPassCache();
+            };
+        }
+
+        void ClearCustomPassCache()
+        {
+            // When custom passes are re-ordered, a topological change happens in the SerializedProperties
+            // So we have to rebuild all the drawers that were storing SerializedProperties.
+            // customPassDrawers.Clear();
```

Comment:
Shouldn't this line be uncommented? Thank you!

Reply:
Yes! I'm not sure what happened there ^^'
Thanks!